diff -Naur audit-2.4.2/bindings/python/Makefile.am audit-2.4.2.tpg/bindings/python/Makefile.am
--- audit-2.4.2/bindings/python/Makefile.am	2015-04-28 20:03:10.000000000 +0000
+++ audit-2.4.2.tpg/bindings/python/Makefile.am	2015-05-24 07:49:51.503029130 +0000
@@ -27,5 +27,6 @@
 
 auparse_la_SOURCES = auparse_python.c
 auparse_la_CPPFLAGS = -I$(top_srcdir)/auparse $(AM_CPPFLAGS) -I@PYINCLUDEDIR@ -fno-strict-aliasing
-auparse_la_LDFLAGS = -module -avoid-version -Wl,-z,relro
+auparse_la_CFLAGS = -shared
+auparse_la_LDFLAGS = -module -avoid-version -shared -Wl,-z,relro
 auparse_la_LIBADD = ${top_builddir}/auparse/libauparse.la ${top_builddir}/lib/libaudit.la
diff -Naur audit-2.4.2/swig/Makefile.am audit-2.4.2.tpg/swig/Makefile.am
--- audit-2.4.2/swig/Makefile.am	2015-04-28 20:03:18.000000000 +0000
+++ audit-2.4.2.tpg/swig/Makefile.am	2015-05-24 07:50:23.861913479 +0000
@@ -27,7 +27,8 @@
 pyexec_PYTHON = audit.py
 pyexec_LTLIBRARIES = _audit.la
 pyexec_SOLIBRARIES = _audit.so
-_audit_la_LDFLAGS = -module -avoid-version -Wl,-z,relro
+_audit_la_CFLAGS = -shared
+_audit_la_LDFLAGS = -module -avoid-version -shared -Wl,-z,relro
 _audit_la_HEADERS: $(top_builddir)/config.h 
 _audit_la_DEPENDENCIES =${top_srcdir}/lib/libaudit.h ${top_builddir}/lib/libaudit.la
 nodist__audit_la_SOURCES  = audit_wrap.c
